services/youtube/youtube-base.js

```diff
@@ -24,6 +24,12 @@ const schema = Joi.object({
         Joi.object({
           viewCount: nonNegativeInteger,
           subscriberCount: nonNegativeInteger,
+        }),
+        Joi.object({
+          viewCount: nonNegativeInteger,
+          likeCount: nonNegativeInteger,
+          commentCount: nonNegativeInteger,
+          favoriteCount: nonNegativeInteger,
```

Comment:
As far as I know we don't actually need the `favoriteCount` value for any of our badges, so instead of adding what is essentially a duplicate schema for `statistics`, let's instead make the `dislikeCount` field optional by changing from `nonNegativeInteger` to `optionalNonNegativeInteger` (which can be imported from the same validators module)